osf_tests/conftest.py

```diff
@@ -37,13 +35,3 @@ def request_context(app):
     context.push()
     yield context
     context.pop()
-
-DISCONNECTED_SIGNALS = {
```

Comment:
This is 100% critical to testing notifications these signals must operate in all environments, reconnecting these signals gives the tests such more integrity.